test.sh

```diff
@@ -26,16 +26,14 @@ python --version
 # Install dependencies.
 pip install --upgrade pip setuptools wheel
 pip install flake8 pytest-xdist pytype pylint pylint-exit
-pip install -r requirements/requirements.txt
-pip install -r requirements/requirements-test.txt
-pip install -r requirements/requirements-examples.txt
+pip install -e ."[test, examples]"
```

Comment:
```suggestion
pip install -e ".[test,examples]"
```

Reply:
Sounds good, Addressed in [46d8a48](https://github.com/deepmind/optax/pull/513/commits/46d8a48f87403bed9b7fb58ad3dd79e1ed4b18c8)